Makefile

```diff
@@ -0,0 +1,2 @@
+test:
```

Comment:
nice to see the makefile here for running these tests. Could consider having it setup a venv for testing purposes as well

Reply:
Excellent point, thank you!